bc_obps/registration/tests/endpoints/test_operator_endpoints.py

```diff
@@ -153,14 +138,26 @@ def test_put_approve_operator(self):
             'cas_admin',
             self.content_type,
             {"status": Operator.Statuses.APPROVED, 'user_operator_id': user_operator.id},
-            custom_reverse_lazy('update_operator', kwargs={'operator_id': operator.id}),
+            custom_reverse_lazy('update_operator_status', kwargs={'operator_id': operator.id}),
         )
 
         assert response.status_code == 200
-
-        assert response.json().get('status') == Operator.Statuses.APPROVED
-        assert response.json().get('is_new') == False
-        assert response.json().get("verified_by") == str(self.user.user_guid)
+        response_dict = response.json()
+        assert response_dict.get('status') == Operator.Statuses.APPROVED
+        assert response_dict.get('is_new') == False
+        assert response_dict.get("verified_by") == str(self.user.user_guid)
+        operator.refresh_from_db()  # refresh the operator object to get the updated status
+        for key in response_dict.keys():
+            if key not in AUDIT_FIELDS:
+                if key == "id":
+                    assert response_dict[key] == str(operator.id)  # String representation of UUID
+                elif key == "verified_at":
+                    # tweaking time format to match the expected format
+                    assert response_dict[key] == operator.verified_at.strftime("%Y-%m-%dT%H:%M:%S.%fZ")[:-4] + "Z"
+                elif key == "verified_by":
+                    assert response_dict[key] == str(operator.verified_by.user_guid)
```

Comment:
This (and some of the other loop assertions) are a bit repetitive of the initial assertions (e.g. `assert response_dict.get("verified_by") == str(self.user.user_guid)`). Do you think we need to check that the response and the db have the same values and that the response has the expected value? In most tests I think we just check one or the other

Reply:
Not needed, I just forgot to remove them :) Thank you!